src/components/forms/DatePicker/Calendar.tsx

```diff
@@ -0,0 +1,327 @@
+import React, { useState, useRef, useEffect } from 'react'
+
+import {
+  DAY_OF_WEEK_LABELS,
+  DAY_OF_WEEK_SHORT_LABELS,
+  MONTH_LABELS,
+} from './constants'
```

Comment:
Can you please surface these as optional props for i18n purposes?

Reply:
Issue added: https://github.com/trussworks/react-uswds/issues/866